docs/v2/10_webgl-browser.md

```diff
@@ -98,6 +98,8 @@ public class WebGLGetGasLimit : MonoBehaviour
 
 ### Transaction Status {#transaction-status}
 
+Allows the user to send a transaction to a specified Ethereum account using the `Web3GL` library. The transaction includes the recipient address, `value` (in wei), `gasLimit`, and `gasPrice`. Once the transaction is sent, the script checks the status of the transaction by calling the `GetTransactionReceipt` function. If the transaction was successful, the script logs a message indicating success, otherwise it logs a message indicating failure. If an exception is thrown during the transaction process, the script logs the exception.
```

Comment:
Since this section is about transaction status, just speak about transaction status. More words are not always better.

This section could read something like: the GetTxRecpt method can be used to await the status of a submitted transaction. If successful, ...; if failure, .... etc., etc,